gandi_tf/main.py

```diff
@@ -25,6 +23,55 @@ def fetch_records(domain):
     return r.text
 
 
+def fetch_domains_list(organization_id):
+    if organization_id is not None:
+        fh_url = f"https://api.gandi.net/v5/domain/domains?per_page=1&sort_by=fqdn&sharing_id={organization_id}"
```

Comment:
Please use the functionality of the requests package to send parameters to an URL: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls